lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3392,6 +3393,9 @@ LogicalResult StmtEmitter::visitStmt(TypedeclOp op) {
     emitError(op, "SV attributes emission is unimplemented for the op");
 
   startStatement();
+  if (isZeroBitType(op.getType()))
+    ps << "// ";
```

Comment:
For comments, should wrap this in a neverbox.  See `visitStmt(Output op)` and other places.  :/.

Concern is if this wraps it won't still be commented-out.

Reply:
Is this the right approach?
```c++
  if (isZeroBitType(op.getType()))
    ps << PP::neverbox << "// ";
```

Reply:
Oh yeah I see the same exact code elsewhere, thanks!